usaspending_api/download/filestreaming/csv_generation.py

```diff
@@ -175,13 +180,16 @@ def parse_source(source, columns, download_job, working_dir, start_time, message
 
 def split_and_zip_csvs(zipfile_path, source_path, source_name, download_job=None):
     try:
-        # Split CSV into separate files
+        # e.g. `Assistance_prime_transactions_delta_%s.csv`
```

Comment:
This comment is out of context. We suggest keeping it `Split CSV into separate files` and copying it above when we're defining source name